client-tez/src/test/java/org/apache/tez/runtime/library/common/sort/buffer/WriteBufferManagerTest.java

```diff
@@ -519,7 +519,14 @@ public Roaring64NavigableMap getShuffleResult(
       return null;
     }
 
-    @Override
+    public Roaring64NavigableMap getShuffleResultForMultiPart(
```

Comment:
This method seems unused.

Reply:
I tryed to delete this method, it will reduce compile error. 
Because MockShuffleWriteClient must implements this method.